src/App.js

```diff
@@ -101,6 +102,14 @@ function App() {
           }
         />
 
+        <Route
+          path="/profile/recommendations"
+          element={
+            <RequireAuth>
+              <RecommendationPage userId={user ? user.userId : null} />
```

Comment:
If you do a `console.log(user)` , you'll see that the object is actually nested and the property for the id is under `_id`:

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/551a6aea-9c41-4406-a78c-ded79dfa7a2f)

so if we're keeping this call here instead of moving it to inside the `RecommendationPage` component, then this would need to be:

```suggestion
              <RecommendationPage userId={user ? user.user._id : null} />
```

Reply:
Akos in the previous commit I have deleted this route from App.js. And actually the now user is not used. Should I delete it or refactor it in some way?

Reply:
We still need the route in `App.js` otherwise the app will not know what to show on: http://localhost:3000/profile/recommendations

![](https://github.com/Code-the-Dream-School/dd-prac-team4-front/assets/9094098/a4cda384-9175-4005-84b1-9ceec887da0e)

Reply:
yeah added 🙈 , thank you , commit b7c70b6b4b564cfa8198230d5f3378742bf489c3